### PR TITLE
Fix crash in LocalizerTask

### PR DESCRIPTION
**Pull request by kurganec**
In gradle a plugin dependency may be declared in one project, but applied in another one. 

In this case LocalizerTask may crash, because buildscript may not have `org.jvnet.localizer:maven-localizer-plugin` on the classpath.

```
Example:
build.gradle -   dependencies { classpath 'org.jenkins-ci.tools:gradle-jpi-plugin:0.15.0' }
my_plugin/build.gradle - apply plugin: 'org.jenkins-ci.jpi'
```
For more information see `JpiLocalizerTaskIntegrationTest`.

In order to fix the problem I've rewritten `LocalizerTask` to call `org.jvnet.localizer.GeneratorTask` directly without `IsolatedAntBuilder`.

This PR contains:
- An integration test to demonstrate the problem
- Fix for the problem as a second commit
- I've changed `org.jvnet.localizer:maven-localizer-plugin` version to 1.12 because 1.13 is not available on maven central.
- I've updated gradle wrapper to version 2.8 to be able to create integration tests.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -19,7 +19,7 @@ sourceCompatibility = '1.6'
 
 dependencies {
     compile gradleApi()
-    compile 'org.jvnet.localizer:maven-localizer-plugin:1.13'
+    compile 'org.jvnet.localizer:maven-localizer-plugin:1.12'
```
Comment (daspilker) [inline build.gradle]: do not downgrade the version, the artifact is in http://repo.jenkins-ci.org/releases/ which is the official repo for all Jenkins related artifacts

File: gradlew [hunk 2]
```diff
@@ -7,7 +7,7 @@
 ##############################################################################
 
 # Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-DEFAULT_JVM_OPTS="-XX:MaxPermSize=256m"
+DEFAULT_JVM_OPTS=""
```
Comment (kurganec) [inline gradlew]: I should probably return this option back. Should I?

Comment (daspilker) [inline gradlew]: Yes

File: src/test/groovy/org/jenkinsci/gradle/plugins/jpi/localizer/JpiLocalizerTaskIntegrationSpec.groovy [hunk 3]
```diff
@@ -0,0 +1,87 @@
+package org.jenkinsci.gradle.plugins.jpi.localizer
```
Comment (daspilker) [inline src/test/groovy/org/jenkinsci/gradle/plugins/jpi/localizer/JpiLocalizerTaskIntegrationSpec.groovy]: do not add a localizer package

File: src/test/groovy/org/jenkinsci/gradle/plugins/jpi/localizer/JpiLocalizerTaskIntegrationSpec.groovy [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+    def 'single-module project should be able to run LocalizerTask'() {
+        given:
+        prepareFile('build.gradle') << """
+        buildscript {
+            repositories {
+                mavenCentral()
+            }
+            dependencies {
+                classpath files(${pluginClasspath})
```
Comment (daspilker) [inline src/test/groovy/org/jenkinsci/gradle/plugins/jpi/localizer/JpiLocalizerTaskIntegrationSpec.groovy]: use `withPluginClasspath` instead, see https://docs.gradle.org/current/javadoc/org/gradle/testkit/runner/GradleRunner.html#withPluginClasspath%28java.lang.Iterable%29

File: gradle/wrapper/gradle-wrapper.properties [hunk 5]
```diff
@@ -1,6 +1,6 @@
-#Wed Feb 17 17:32:27 CET 2016
+#Sun Mar 06 13:55:02 MSK 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.5-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
```
Comment (daspilker) [inline gradle/wrapper/gradle-wrapper.properties]: use the `-all` distribution

Comment (daspilker) [inline gradle/wrapper/gradle-wrapper.properties]: also update the `wrapper` task in `build.gradle`

**Comment by daspilker** [comment]
OK, cool. Gradle 2.13 will provide a better TestKit, which will not require so much boilerplate code. See https://github.com/gradle/gradle/commit/5be4cd00a3db1f36f735316008bd7cd826a5c02d#diff-5c5d96f87fb425d3fe3bc2400a400a4c. But for now, it's OK. Can you squash your commits and force push?

**Pull request merged**